.gitignore

```diff
@@ -73,3 +73,6 @@ target/
 
 *.kdev4
 *.kate-swp
+
+# Virtual environments
+venv*/
```

Comment:
Please not.  I'd rather get rid of random peoples' IDE clutter landing on the maintained ignore list.  Git also has a local ignore function if you want to make sure it's not shown in status.  No need to assume that everyone wants to ignore any directory named like `venv*`.

Reply:
Alright, fair. It _is_ rather standard and many projects choses to, but it's not PEP-ed, so its ok not to. -- But then I would suggest we remove the other IDE excludes as well as you point out.